src/app/loadout-builder/filter/ExoticTile.m.scss

```diff
@@ -15,6 +15,10 @@
   }
 }
 
+.selected {
+  border-color: orange;
```

Comment:
Can you use $orange?

Reply:
Sure, found the usage in a few places so I changed them all.